tsconfig-schema.json

```diff
@@ -6,17 +6,35 @@
         "plugins": {
           "items": {
             "properties": {
-              "transform": { "type": "string" },
-              "import": { "type": "string" },
-              "after": { "type": "boolean" },
-              "afterDeclarations": { "type": "boolean" },
+              "transform": {
+                "type": "string"
+              },
+              "import": {
+                "type": "string"
+              },
+              "after": {
+                "type": "boolean"
+              },
+              "afterDeclarations": {
+                "type": "boolean"
+              },
               "type": {
                 "oneOf": [
-                  { "const": "program" },
-                  { "const": "config" },
-                  { "const": "checker" },
-                  { "const": "raw" },
-                  { "const": "compilerOptions" }
+                  {
+                    "const": "program"
+                  },
+                  {
+                    "const": "config"
+                  },
+                  {
+                    "const": "checker"
+                  },
+                  {
+                    "const": "raw"
+                  },
+                  {
+                    "const": "compilerOptions"
+                  }
```

Comment:
I don't think we actually need that part, at least not for VS Code support.
VS Code automatically merges our Schema with Typescript's.

Reply:
Removed in a9440e48